### DISCUSSION TITLE
Change in albumin concentration

### DISCUSSION THREAD

**Discussion by AParisotto** [Open-Systems-Pharmacology/Forum#850] [Questions & Problems] [answered]
Hi everyone

How can I change the concentration of albumin levels in PK-SIM?
In an already published study the value of albumin concentration in a group of patients is 38 g / L and I would like to add this value to the virtual individuals of PK-SIM.

Thanks

**Comment by Ricardo-DO** [comment, marked as answer]
Hi 

For what I know, when you create your individual you can change the scale factor for the protein content in plasma.

![albumin](https://user-images.githubusercontent.com/59565610/98832718-0e7ef500-2435-11eb-94fd-aa7297d38fc0.png)

I think that the reference value is around 4 g/100 mL, so I think your individuals should be alright with the default values.

Regards,
Ricardo
